emotes/command_operations.py

```diff
@@ -2,17 +2,41 @@
 import json
 import discord
 from shared.constants import CONTRIBUTORS_FILE_PATH
+from shared.helpers import get_guild_member_check_role
 from typing import List, Dict, Any
 
+"""
+emotes/command_operations.py is responsible for handling the business logic associated with emote commands
+This includes adding, removing, listing, and direct messaging contributors.
+"""
```

Comment:
No worries here but just FYI this is usually done at the very top

Reply:
> No worries here but just FYI this is usually done at the very top

Went through and moves these to the top